pkgs/os-specific/linux/open-iscsi/kmod-via-make.diff

```diff
@@ -0,0 +1,14 @@
+--- open-iscsi/usr/Makefile.orig	2016-04-24 13:51:26.301726559 +0200
```

Comment:
You are right, this works equally well. I removed the patch. I guess, I went for a patch to make it more upstreamable.